lib/utils/import.js

```diff
@@ -49,3 +50,33 @@ function getSourceModuleName(node) {
     return undefined;
   }
 }
+
+/**
+ * Gets an import identifier (either imported or local name) from the specified ImportDeclaration.
+ *
+ * @param {node} node the ImportDeclaration to find the import identifier for
+ * @param {string} source the source, or module name string, of the import
+ * @param {string} [namedImportIdentifier=null] the named import identifier to find (will return the alias of the import, of found)
+ * @returns {null} if no import is found with that name
+ */
+function getImportIdentifier(node, source, namedImportIdentifier = null) {
+  assert(
+    isImportDeclaration(node),
+    `getImportIdentifier should be called with a node that's type is 'ImportDeclaration'. You passed '${node.type}'`
+  );
+
+  if (node.source.value !== source) {
+    return null;
+  }
+
+  return node.specifiers
+    .filter(specifier => {
+      return (
+        (specifier.type === 'ImportSpecifier' &&
+          specifier.imported.name === namedImportIdentifier) ||
+        specifier.type === 'ImportDefaultSpecifier'
```

Comment:
```suggestion
        (!namedImportIdentifier && specifier.type === 'ImportDefaultSpecifier')
```

And can you add this test case to verify this fix:
```
it('returns null when no default import is found', () => {
  const node = babelEslint.parse("import { later } from '@ember/runloop';").body[0];
  const identifier = importUtils.getImportIdentifier(node, '@ember/runloop');

  expect(identifier).toBeNull();
});
```